src/app/api/v1/premium/route.ts

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+export async function GET(req: NextRequest) {
+  try {
+    const { searchParams } = new URL(req.url);
+    const email = searchParams.get("email");
+    console.debug("Subscription redirect for:", email);
```

Comment:
Q: since this is an API route, where does this get logged? Terminal? Log file? And is this some PII risk, or was it just meant for local debuggery?

Reply:
Local just for demonstrative purposes, I'll just remove it, good catch thanks!